v3/src/components/graph/adornments/components/movable-line.scss

```diff
@@ -19,15 +26,15 @@
 }
 
 .movable-line-lower-cover {
-  cursor: url("../../../assets/LinePivotBotLeft.png") 8 8, grab;
+  cursor: url("../../../../assets/LinePivotBotLeft.png") 8 8, grab;
```

Comment:
I'm not sure it's valid, but the Movable Value component's [stylesheet references LinePivotBotLeft.png](url) which makes me wonder if these PNGs really do belong in the more general `v3/src/assets` directory. Do you happen to know if that reference in Movable Value's stylesheet is a copy-paste error or something like that? For some reason, I'm having a hard time finding the Movable Value adornment in V2 to see if it uses the graphic there. Even if it _does_ actually use it, do you think it would make sense to move the other two PNGs that are only used by Movable Line to `v3/src/components/graph/adornments/movable-line`?

Reply:
It's a judgment call. If we're thinking of them as potentially reusable elements they should probably have more generic names (e.g. `square-drag-handle.png`) and be in the shared `assets` folder. If we're thinking of them as movable-line-specific then their movable-line-specific names are appropriate and it seems like they could be in the movable line folder. Ultimately, it's about what will make the most sense to future engineers trying to make sense of the code.